modules/globebrowsing/tile/tileprovider/singleimageprovider.cpp

```diff
@@ -78,16 +74,19 @@ void SingleImageProvider::update() {
 }
 
 void SingleImageProvider::reset() {
-    auto tileTexture = std::shared_ptr<ghoul::opengl::Texture>(
-        ghoul::io::TextureReader::ref().loadTexture(_imagePath)
-    );
-    auto tileStatus = tileTexture != nullptr ? Tile::Status::OK : Tile::Status::IOError;
+    _tileTexture = ghoul::io::TextureReader::ref().loadTexture(_imagePath);
+    auto tileStatus = _tileTexture ? Tile::Status::OK : Tile::Status::IOError;
```

Comment:
Haha yes! That is confusing. `auto tileMetaData = nullptr` is an interesting one :P